integration-app/src/main/java/uk/gov/crowncommercial/esourcing/integration/exception/SalesforceUpdateException.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.crowncommercial.esourcing.integration.exception;
+
+import java.util.Map;
+
+public class SalesforceUpdateException extends RuntimeException {
+
+  private static final long serialVersionUID = 1L;
+
+  private final Map<String, String> map;
```

Comment:
This is ok - I personally would have created an object that contains the procurement ref and email address and then sent a list of those objects. That is more type safe and when you want to add in a third parameter then extendable more easily.

Reply:
Done